app/scaffold-controllers/navigationController.js

```diff
@@ -149,5 +192,8 @@ function(
         return this.isActive;
     };
 
+    NavigationController.prototype.setActive = function(state) {
```

Comment:
Spoke with @lizcross and the original way of going about flipping state (direct access) is okay. We are avoiding creating `activate`/`deactivate` methods because it implies that we are doing something more than flipping a state.